system-tests/projects/create-react-app-configured-bundled-mount/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
This looks like the README that CRA creates - can we remove it for this system test, or better yet, include a brief explanation of what this project is testing? a'la

https://github.com/cypress-io/cypress/tree/10.0-release/system-tests/projects/react-a11y
or
https://github.com/cypress-io/cypress/tree/10.0-release/system-tests/projects/migration-e2e-no-plugins-support-file.

Reply:
Same question with a lot of files in this repo - can we remove the react logo, cut down the dependency list (eg, remove testing-library), remove App.css and otherwise cut this down to show what's trying to be tested more precisely.

Reply:
You're spot-on, but there's already two big PRs in progress to clean up the duplication of system tests by Tim and Lachlan... so I'm not touching that infrastructure right now.

Reply:
I don't mean the infrastructure, I mean the new files added directly here in this PR.